charts/hivemq-platform/templates/_helpers.tpl

```diff
@@ -475,6 +474,82 @@ Usage: {{- include "hivemq-platform.validate-additional-volumes" . }}
 {{- end -}}
 {{- end -}}
 
+{{/*
+Validates the broker license, additional broker licenses, the enterprise extension licenses and the Data Hub license so:
+ - When the `license.create` value is enabled, a least one license content is defined.
+ - Only one of `data` or `overrideLicense` are set, and not both.
+ - No duplicated "license.lic" Secret entries is found for additional broker licenses.
+Usage: {{ include "hivemq-platform.validate-licenses" . }}
+*/}}
+{{- define "hivemq-platform.validate-licenses" -}}
+{{- $licenseContentExists := "" -}}
+{{- if .Values.license.create -}}
+    {{- include "hivemq-platform.validate-license-content" (dict "licenses" .Values.license.additionalLicenses "licenseLabel" "Additional HiveMQ Broker") -}}
+    {{- include "hivemq-platform.validate-license-content" (dict "licenses" .Values.license.extensions "licenseLabel" "HiveMQ Enterprise Extension") -}}
+    {{- include "hivemq-platform.validate-license-content" (dict "licenses" .Values.license.dataHub "licenseLabel" "HiveMQ Data Hub") -}}
+    {{- if and (not .Values.license.overrideLicense) (not .Values.license.data) (not .Values.license.additionalLicenses) (not .Values.license.extensions) (not .Values.license.dataHub) -}}
+        {{- fail (printf "HiveMQ Platform license content cannot be empty") -}}
+    {{- end -}}
+    {{- if and .Values.license.data .Values.license.overrideLicense -}}
+        {{- fail (printf "Both `data` and `overrideLicense` values are set for the HiveMQ Broker license content. Please, use only one of them") -}}
+    {{- end -}}
+    {{- include "hivemq-platform.validate-duplicated-additional-broker-license" . -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Validates the licenses content within a range function:
+ - At least either `data` or `overrideLicense` is not empty.
+ - Only one of `data` or `overrideLicense` are set, and not both.
+Usage: {{ include "hivemq-platform.validate-license-content" (dict "licenses" .licenses "licenseLabel" "HiveMQ Enterprise Extension") }}
+*/}}
+{{- define "hivemq-platform.validate-license-content" -}}
+{{- $label := .licenseLabel -}}
+{{- range $licenseName, $license := .licenses -}}
+    {{- if and (not $license.data) (not $license.overrideLicense) -}}
+        {{- fail (printf "%s '%s' license content cannot be empty. Please, use either `data` or `overrideLicense` values" $label $licenseName) }}
+    {{- end -}}
+    {{- if and (gt (len $license) 1) (hasKey $license "data") (hasKey $license "overrideLicense") -}}
+        {{- fail (printf "Both `data` and `overrideLicense` values are set for the %s '%s' license content. Please, use only one of them" $label $licenseName) }}
+    {{- else if gt (len $license) 1 -}}
+        {{- fail (printf "Invalid values for setting a license content. Only `data` or `overrideLicense` values are allowed") }}
```

Comment:
The error message doesn't use the `$label` and `$licenseName`

Reply:
On line 513, it uses both `$label` and `$licenseName`: 
```
...the %s '%s'...
```

Or not sure if you're referring to line 515 where there is no variable used

Reply:
The comment was for the `Invalid values for setting a license content` error message.